afids_utils/tests/test_io.py

```diff
@@ -64,6 +64,30 @@ def test_invalid_template(self, afids_coords: NDArray[np.single]) -> None:
                 "/invalid/fcsv/path",
             )
 
+    @given(afids_coords=afid_coords(bad_range=True))
+    def test_invalid_num_afids(self, afids_coords: NDArray[np.single]) -> None:
+        out_fcsv_file = tempfile.NamedTemporaryFile(
```

Comment:
I think I would either do one of the following here and in the other added test to make sure the temp file gets deleted:

- Use `NamedTemporaryFile` as a context manager without the `delete=False`
- Use `TemporaryDirectory` instead and just let `afids_to_fcsv` handle making the file

Reply:
Went with the first option here. I kept it in the one test where the out file is explicitly removed.